yarn.lock

```diff
@@ -4578,16 +4578,16 @@ csstype@^2.2.0:
   resolved "https://registry.yarnpkg.com/csstype/-/csstype-2.5.7.tgz#bf9235d5872141eccfb2d16d82993c6b149179ff"
   integrity sha512-Nt5VDyOTIIV4/nRFswoCKps1R5CD1hkiyjBE9/thNaNZILLEviVw9yWQw15+O+CpNjQKB/uvdcxFFOrSflY3Yw==
 
-custom-functions-metadata@^0.0.19:
-  version "0.0.19"
-  resolved "https://registry.yarnpkg.com/custom-functions-metadata/-/custom-functions-metadata-0.0.19.tgz#1063e0c50a021e70456a5e6169a602ddfbc31fdf"
-  integrity sha512-uhBIgDzOcSjPdZ9sT+OVJ2x/eSBYvzOylUxoUb3SZ+W7TZTvFhwOo647TuBt2E4bq/4SoF8zPpie6M61YTYwcw==
+custom-functions-metadata@0.0.22:
```

Comment:
Likewise, this line would have ended up with `^` in it as well:
`custom-functions-metadata@^0.0.22:`

Reply:
It's also the reason we should update @types/office.js to be something higher than 0.0.170. You can't automatically get updates. I tried using ">= 0.0.170" instead but it doesn't stick. So we have to keep manually applying updates to get the latest in https://github.com/OfficeDev/Office-Addin-TaskPane.